samples/assistant/csharp-ooproc/AssistantSampleIsolated/AssistantSampleIsolated.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <AzureFunctionsVersion>v4</AzureFunctionsVersion>
+    <OutputType>Exe</OutputType>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Include="..\..\README.md" Link="README.md" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.20.1" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore" Version="1.2.0" />
```

Comment:
Do we still need this `AspNetCore` package now that we're not using the ASP.NET types anymore?

Reply:
That's a fair point. I'll open up a new work item to change the samples to use ASP .NET types and use ASP .NET core integration once the fix has been shipped on the host